examples/imgui_impl_metal.mm

```diff
@@ -23,8 +23,17 @@
 #include "imgui_impl_metal.h"
 
 #import <Metal/Metal.h>
-// #import <QuartzCore/CAMetalLayer.h> // Not supported in XCode 9.2. Maybe a macro to detect the SDK version can be used (something like #if MACOS_SDK >= 10.13 ...)
+#import <QuartzCore/CAMetalLayer.h>
 #import <simd/simd.h>
```

Comment:
Could you clarify why this header needs to be added back?
See comment:
https://github.com/ocornut/imgui/pull/1929#issuecomment-430092470

Thanks!

Reply:
But it must be have CAMetalLayer to create multi-viewport.
So I don't know what happen in Xcode 9.2, I think someone bulid it in simulator (Simulator haven't METAL API)

Reply:
I pushed the code that it can compile and run.
It can be also compile that disable the macro #if TARGET_OS_SIMULATOR == 0 || __IPHONE_13_0 to #if 0.